pkg/crc/oc/oc_cache.go

```diff
@@ -43,6 +40,10 @@ func (oc *OcCached) IsCached() bool {
 	return true
 }
 
+func ocFileFilter(filename string) bool {
+	return filepath.Base(filename) == constants.OcBinaryName
```

Comment:
Is this returning `true` when the bundle matches the executable name?
if so, why is this called ocFileFilter? why not refering to the fact the binaryname is checked?
And how about `oc.exe` on Windows?

Reply:
> Is this returning true when the bundle matches the executable name?

yes

> if so, why is this called ocFileFilter? why not refering to the fact the binaryname is checked?

Call it `matchOcBinaryName` then?

> And how about oc.exe on Windows?

`pkg/crc/constants/constants_windows.go: OcBinaryName = "oc.exe"`